src/lib/blocks.tsx

```diff
@@ -93,6 +133,23 @@ const replaceElementWithBlock = (element: HTMLElement, component: JSX.Element):
   )
 }
 
+export const renderCTABlocks = (element: HTMLElement): void => {
```

Comment:
Note: Depending on merge order, this should be rewritten in this PR or [this PR](https://github.com/openstax/k12-apps-raise/pull/10) to use the `renderContentBlocksByClass` utility function in the latter.

Reply:
Waiting for your PR to be merged for the time being.